### DISCUSSION TITLE
Does iModel/Plantsight support Normal Mapping

### DISCUSSION THREAD

**Discussion by PeterJBassett** [iTwin/itwinjs-core#3931] [Q&A] [answered]
Hello - the FutureOn to iTwin connector is being used to convert FutureOn subsea models into iModels for display in Plantsight.

Does iModel/Plantsight visualisation have the capability to perform 'normal mapping' when rendering a surface.
See https://en.wikipedia.org/wiki/Normal_mapping

If iModel does support normal mapping, please can you point me to the imodeljs library module calls to use, and/or documentation or examples.

**Comment by pmconne** [comment]
iTwin.js visualization does not currently support normal maps.

**Comment by pmconne** [comment, marked as answer]
We are actively looking into adding support for normal maps. Closing this question as answered - please follow #3989 for updates on the enhancement.
